charts/enterprise-kyverno-operator/README.md

```diff
@@ -32,28 +32,28 @@ Additional parameters corresponding to custom CA or HTTP proxies, NO_PROXY shoul
 
 View various Resources created
 ```bash
-kubectl -n enterprise-kyverno-operator get kyvernoes.security.nirmata.io #(CR that defines kyverno settings)
-kubectl -n enterprise-kyverno-operator get policysets.security.nirmata.io #(CRs corresponding to default policysets installed)
+kubectl -n nirmata-system get kyvernoconfigs.security.nirmata.io #(CR that defines kyverno settings)
+kubectl -n nirmata-system get policysets.security.nirmata.io #(CRs corresponding to default policysets installed)
 
 kubectl -n kyverno get po #(should show Kyverno pods getting ready)
 kubectl get cpol #(should show policies installed by initial policysets)
 ```
 
 If you need to modify Kyverno configuration, change CR directly or via Helm Upgrade
 ```bash
-kubectl -n enterprise-kyverno-operator edit kyvernoes.security.nirmata.io kyverno (and set replicas to 3)
+kubectl -n nirmata-system edit kyvernoconfigs.security.nirmata.io kyverno (and set replicas to 3)
 
-helm upgrade enterprise-kyverno-operator nirmata/enterprise-kyverno-operator -n enterprise-kyverno-operator --create-namespace --set licenseKey=<licenseKey> --set kyverno.replicas=3
+helm upgrade nirmata-kyverno-operator nirmata/nirmata-kyverno-operator -n nirmata-system --create-namespace --set licenseKey=<licenseKey> --set kyverno.replicas=3
 ```
 
 Removing a Policy Set CR removes policies contained in it
 ```bash
-kubectl -n enterprise-kyverno-operator delete policysets best-practices
+kubectl -n nirmata-system delete policysets best-practices
 ```
 
 To remove Enterprise Kyverno and components
```

Comment:
Should this be `To remove Nirmata Kyverno Operator and components` instead of `To remove Enterprise Kyverno and components`
May be we have to do this elsewhere in the README

Reply:
Let me confirm with Anusha. There are 2-3 places where we call operator as Enterprise Kyverno (including the chart intro itself.